Core/include/Acts/Vertexing/TrackAtVertex.hpp

```diff
@@ -15,45 +15,67 @@
 
 namespace Acts {
 
-/// @class TrackAtVertex
+/// @struct FittedMomentum
+///
+/// @brief Vertex fitters return a vertex position and updated track momenta at said position (+corresponding covariances). The updated track momenta and their covariances are saved in the following struct.
+struct FittedMomentum {
+  FittedMomentum(Vector3 mom, ActsSymMatrix<3> cov)
```

Comment:
```suggestion
  FittedMomentum(const Vector3& mom, const ActsSymMatrix<3>& cov)
```

we cannot `std::move` those so getting a `const &` is avoiding a second copy